src/server_api.ts

```diff
@@ -285,29 +302,53 @@ export namespace ServerApi {
     | BeginSponsoringFutureReservesOperationRecord
     | EndSponsoringFutureReservesOperationRecord
     | RevokeSponsorshipOperationRecord;
-  export interface TradeRecord extends Horizon.BaseResponse {
-    id: string;
-    paging_token: string;
-    ledger_close_time: string;
-    offer_id: string;
-    base_offer_id: string;
-    base_account: string;
-    base_amount: string;
-    base_asset_type: string;
-    base_asset_code?: string;
-    base_asset_issuer?: string;
-    counter_offer_id: string;
-    counter_account: string;
-    counter_amount: string;
-    counter_asset_type: string;
-    counter_asset_code?: string;
-    counter_asset_issuer?: string;
-    base_is_seller: boolean;
 
-    base: CallFunction<AccountRecord>;
-    counter: CallFunction<AccountRecord>;
-    operation: CallFunction<OperationRecord>;
+  export namespace TradeRecord {
+    interface Base extends Horizon.BaseResponse {
+      id: string;
+      paging_token: string;
+      ledger_close_time: string;
+      offer_id: string;
```

Comment:
`offer_id` has been deprecated for a while so we plan to remove it from the API in the AMM release

Reply:
It appears it's still in the trade response? https://github.com/stellar/go/blob/amm/protocols/horizon/main.go#L368

Reply:
Resolved offline: it _will_ be removed.